Automattic/delphin
# Remove About page

This pull request removes the `About` page, which is no longer necessary.
#### Testing instructions
1. Run `git checkout remove/about` and start your server, or open a [live branch](https://delphin.live/?branch=remove/about)
2. Open the [`About` page](http://delphin.localhost:1337/about)
3. Check that you are presented with the 404 error page
#### Reviews

@Automattic/sdev-feed
